R/remote.R

```diff
@@ -247,7 +247,9 @@ orderly_run_remote <- function(name, parameters = NULL, ref = NULL,
                                progress = TRUE,
                                root = NULL, locate = TRUE, instance = NULL,
                                remote = NULL) {
-  remote <- get_remote(remote, orderly_config(root, locate))
+  cfg <- orderly_config(root, locate)
+  git_ref_is_valid(ref, cfg$root)
```

Comment:
I think at this point we should translate ref from branch to hash, no? So that way we will run the commit that the researcher sees at some point in the future, even if more commits are pushed to that branch

Reply:
Yes, that function does it already for us too. I've simplified this PR a bit